### ISSUE TITLE
[CDRWin] Index 00 incorrectly set in some CUE images

### ISSUE THREAD

**Issue by TheRogueArchivist** [claunia/DiscImageChef#719]
### Version

5.3.0

### Tested debug version?

- [X] Yes

### Which operating systems have you used?

- [ ] Windows
- [ ] Linux
- [ ] macOS
- [ ] Other

### What is the architectural bit size you're using?

- [ ] 32-bit
- [ ] 64-bit
- [ ] Unsure or unknown

### What processor are you using?

- [X] An Intel or AMD
- [ ] An ARM or Apple Silicon
- [ ] Unsure or unknown

### Image format

CDRWin CUE

### Application that created the image

Aaru

### What were you doing when it failed?

- [X] I was converting the image to another format...
- [ ] I was retrieving image information...
- [ ] I was retrieving filesystem information...
- [ ] I was listing the filesystem contents...
- [ ] I was extracting the filesystem contents...
- [ ] I was creating a metadata sidecar...
- [ ] I was checksumming the image...
- [ ] I was verifying the image...
- [ ] I was calculating the image entropy...

### Description

When converting an image from Aaruf to CDRWin CUE (and most likely when dumping straight to CUE, but that hasn't been tested yet), Aaru sets both index 0 and 1 to 00:00:33, and other software such as Daemon Tools isn't able to recognize it. Changing index 0 to 00:00:00 matches dumps created by other software, and allows Daemon Tools to mount the image.

### Exact command line used

Aaru image convert '.\Roger Waters - The Wall - Live in Berlin (USA) (846 611-2) (PDO) (Disc 2) (AD).aaruf' '.\Roger.cue' -f

### Expected behavior

CUE image created by Aaru to work.

### Actual behavior

CUE image created by Aaru doesn't work in other software.

### Output of command execution with debug output enabled

```shell
aaru 5.3.1
Copyright c 2011-2022 Natalia Portillo

DEBUG (Image convert command): --cicm-xml=
DEBUG (Image convert command): --comments=
DEBUG (Image convert command): --count=64
DEBUG (Image convert command): --creator=
DEBUG (Image convert command): --debug=True
DEBUG (Image convert command): --drive-manufacturer=
DEBUG (Image convert command): --drive-model=
DEBUG (Image convert command): --drive-revision=
DEBUG (Image convert command): --drive-serial=
DEBUG (Image convert command): --force=True
DEBUG (Image convert command): --format=
DEBUG (Image convert command): --geometry=
DEBUG (Image convert command): --input=.\Roger Waters - The Wall - Live in Berlin (USA) (846 611-2) (PDO) (Disc 2) (AD).aaruf
DEBUG (Image convert command): --media-barcode=
DEBUG (Image convert command): --media-lastsequence=0
DEBUG (Image convert command): --media-manufacturer=
DEBUG (Image convert command): --media-model=
DEBUG (Image convert command): --media-partnumber=
DEBUG (Image convert command): --media-sequence=0
DEBUG (Image convert command): --media-serial=
DEBUG (Image convert command): --media-title=
DEBUG (Image convert command): --options=
DEBUG (Image convert command): --output=.\Roger.cue
DEBUG (Image convert command): --resume-file=
DEBUG (Image convert command): --verbose=False
DEBUG (Image convert command): --fix-subchannel-position=True
DEBUG (Image convert command): --fix-subchannel=False
DEBUG (Image convert command): --fix-subchannel-crc=False
DEBUG (Image convert command): --generate-subchannels=False
DEBUG (Image convert command): Parsed options:
DEBUG (Format detection): Trying plugin Aaru Format
Input image format identified by Aaru Format.
DEBUG (Aaru Format plugin): Memory snapshot: 6448288 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 6448288 bytes
DEBUG (Aaru Format plugin): Index at 323181035 contains 63 entries
DEBUG (Aaru Format plugin): Memory snapshot: 6448288 bytes
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 104
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 4556787
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 9762544
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 15681906
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 21535706
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 27083086
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 31739051
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 35968841
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 40254227
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 44683423
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 49040458
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 53957562
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 59668212
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 65271544
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 70819183
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 75230775
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 80552991
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 86880411
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 92383079
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 97069334
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 102961802
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 109170419
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 114752265
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 120876812
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 127463499
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 133947142
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 140282279
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 146431766
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 152164118
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 158296158
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 163588005
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 169197461
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 175635041
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 181731889
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 187821312
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 193958428
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 200103281
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 206339719
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 212649087
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 218360791
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 223905657
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 229866721
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 235954543
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 241944299
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 246474560
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 251911148
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 257400398
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 263143059
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 268411426
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 274404105
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 280431701
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 286115963
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 290496343
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 295598281
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 300553208
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 305957094
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 312139653
DEBUG (Aaru Format plugin): Block type DataBlock with data type UserData is indexed to be at 318556338
DEBUG (Aaru Format plugin): Block type DataBlock with data type CompactDiscToc is indexed to be at 322391404
DEBUG (Aaru Format plugin): Block type DumpHardwareBlock with data type NoData is indexed to be at 322391544
DEBUG (Aaru Format plugin): Block type DeDuplicationTable with data type UserData is indexed to be at 322391673
DEBUG (Aaru Format plugin): Block type DataBlock with data type CdSectorSubchannel is indexed to be at 322452736
DEBUG (Aaru Format plugin): Block type TracksBlock with data type NoData is indexed to be at 323180488
DEBUG (Aaru Format plugin): Memory snapshot: 6481056 bytes
DEBUG (Aaru Format plugin): Found data block type CompactDiscToc at position 322391404
DEBUG (Aaru Format plugin): Memory snapshot: 6489248 bytes
DEBUG (Aaru Format plugin): Took 0.009071 seconds to decompress block
DEBUG (Aaru Format plugin): Memory snapshot: 40084696 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 40084696 bytes
DEBUG (Aaru Format plugin): Found dump hardware block at position 322391544
DEBUG (Aaru Format plugin): Memory snapshot: 40084696 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 40092888 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 40092888 bytes
DEBUG (Aaru Format plugin): Decompressing DDT...
DEBUG (Aaru Format plugin): Took 0.0494912 seconds to decompress DDT
DEBUG (Aaru Format plugin): Memory snapshot: 77034856 bytes
DEBUG (Aaru Format plugin): Found data block type CdSectorSubchannel at position 322452736
DEBUG (Aaru Format plugin): Memory snapshot: 77034856 bytes
DEBUG (Aaru Format plugin): Took 73.339ms to de-sequentialize subchannel.
DEBUG (Aaru Format plugin): Took 1142.6619ms to interleave subchannel.
DEBUG (Aaru Format plugin): Took 1216.0009ms to untransform subchannel.
DEBUG (Aaru Format plugin): Took 1.4707332 seconds to decompress block
DEBUG (Aaru Format plugin): Memory snapshot: 214011664 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214011664 bytes
DEBUG (Aaru Format plugin): Found 13 tracks at position 13
DEBUG (Aaru Format plugin): Memory snapshot: 214025200 bytes
DEBUG (Aaru Format plugin): Image created on 1/21/2022 06:53:04
DEBUG (Aaru Format plugin): Image last written on 1/21/2022 07:31:39
DEBUG (Aaru Format plugin): Memory snapshot: 214033344 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214033344 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214041536 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214041536 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214041536 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214049728 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 214049728 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 229754568 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 229754568 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 229761728 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 245497648 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 245505840 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 245505840 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 261017816 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 261017816 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 261017816 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 276520384 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 276520384 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 276520384 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 292925384 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 292933576 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 292933576 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 308702616 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 308702616 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 308702616 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 325517752 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 325525944 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 325525944 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 342705576 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 342705576 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 342713768 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 359508256 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 359508256 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 359508256 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 376580792 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 376588984 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 376588984 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 392202032 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 392202032 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 392202032 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 409328528 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 409328528 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 409328528 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 409328528 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 409328528 bytes
DEBUG (Convert-image command): Correctly opened image file.
DEBUG (Convert-image command): Image without headers is 323178046 bytes.
DEBUG (Convert-image command): Image has 236560 sectors.
DEBUG (Convert-image command): Image identifies media type as CDDA.
Output image format: CDRWin cuesheet.
236560 sectors to convert
Converting sectors 4129 to 4193 in track 1 (1.75% done)DEBUG (Aaru Format plugin): Memory snapshot: 430553288 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 446866408 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 446866408 bytes
Converting sectors 8225 to 8289 in track 1 (3.48% done)DEBUG (Aaru Format plugin): Memory snapshot: 464600088 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 481618616 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 481618616 bytes
Converting sectors 12321 to 12385 in track 1 (5.21% done)DEBUG (Aaru Format plugin): Memory snapshot: 499282808 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 516219392 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 516227584 bytes
Converting sectors 16417 to 16481 in track 1 (6.94% done)DEBUG (Aaru Format plugin): Memory snapshot: 537999208 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 554629368 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 554629368 bytes
Converting sectors 24569 to 24633 in track 2 (10.39% done)DEBUG (Aaru Format plugin): Memory snapshot: 194139864 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 209459128 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 209467320 bytes
Converting sectors 28665 to 28729 in track 2 (12.12% done)DEBUG (Aaru Format plugin): Memory snapshot: 227002680 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 242387040 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 242387040 bytes
Converting sectors 36875 to 36939 in track 3 (15.59% done)DEBUG (Aaru Format plugin): Memory snapshot: 227893096 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 243332720 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 243340912 bytes
Converting sectors 40971 to 41035 in track 3 (17.32% done)DEBUG (Aaru Format plugin): Memory snapshot: 261026856 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 277026552 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 277026552 bytes
Converting sectors 45067 to 45131 in track 3 (19.05% done)DEBUG (Aaru Format plugin): Memory snapshot: 298777696 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 315579128 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 315579128 bytes
Converting sectors 49163 to 49227 in track 3 (20.78% done)DEBUG (Aaru Format plugin): Memory snapshot: 333161352 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 349855464 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 349855464 bytes
Converting sectors 53259 to 53323 in track 3 (22.51% done)DEBUG (Aaru Format plugin): Memory snapshot: 367367224 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 383997456 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 383997456 bytes
Converting sectors 65546 to 65610 in track 5 (27.71% done)DEBUG (Aaru Format plugin): Memory snapshot: 440903816 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 458322080 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 458322080 bytes
Converting sectors 69642 to 69706 in track 5 (29.44% done)DEBUG (Aaru Format plugin): Memory snapshot: 480028136 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 496621776 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 496621776 bytes
Converting sectors 77864 to 77928 in track 6 (32.92% done)DEBUG (Aaru Format plugin): Memory snapshot: 535773392 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 552756832 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 552756832 bytes
Converting sectors 81960 to 82024 in track 6 (34.65% done)DEBUG (Aaru Format plugin): Memory snapshot: 570272712 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 587564112 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 587564112 bytes
Converting sectors 86056 to 86120 in track 6 (36.38% done)DEBUG (Aaru Format plugin): Memory snapshot: 605102360 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 621766992 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 621766992 bytes
Converting sectors 90152 to 90216 in track 6 (38.11% done)DEBUG (Aaru Format plugin): Memory snapshot: 643576736 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 660800416 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 660800416 bytes
Converting sectors 94248 to 94312 in track 6 (39.84% done)DEBUG (Aaru Format plugin): Memory snapshot: 678309128 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 695983104 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 695991296 bytes
Converting sectors 98344 to 98408 in track 6 (41.57% done)DEBUG (Aaru Format plugin): Memory snapshot: 717742168 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 735308592 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 735308592 bytes
Converting sectors 102440 to 102504 in track 6 (43.30% done)DEBUG (Aaru Format plugin): Memory snapshot: 752903432 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 770321352 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 770321352 bytes
Converting sectors 106536 to 106600 in track 6 (45.04% done)DEBUG (Aaru Format plugin): Memory snapshot: 787837208 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 805069480 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 805069480 bytes
Converting sectors 110632 to 110696 in track 6 (46.77% done)DEBUG (Aaru Format plugin): Memory snapshot: 826830728 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 843645864 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 843645864 bytes
Converting sectors 114726 to 114790 in track 7 (48.50% done)DEBUG (Aaru Format plugin): Memory snapshot: 861227416 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 878450432 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 878450432 bytes
Converting sectors 118822 to 118886 in track 7 (50.23% done)DEBUG (Aaru Format plugin): Memory snapshot: 895967168 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 912341800 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 912341800 bytes
Converting sectors 122918 to 122982 in track 7 (51.96% done)DEBUG (Aaru Format plugin): Memory snapshot: 139723688 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 156415736 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 156423928 bytes
Converting sectors 127014 to 127078 in track 7 (53.69% done)DEBUG (Aaru Format plugin): Memory snapshot: 147869280 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 165389448 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 165389448 bytes
Converting sectors 131110 to 131174 in track 7 (55.42% done)DEBUG (Aaru Format plugin): Memory snapshot: 187169480 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 204357112 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 204357112 bytes
Converting sectors 135202 to 135266 in track 8 (57.15% done)DEBUG (Aaru Format plugin): Memory snapshot: 221890072 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 239062088 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 239070280 bytes
Converting sectors 139298 to 139362 in track 8 (58.88% done)DEBUG (Aaru Format plugin): Memory snapshot: 256573848 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 273801744 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 273801744 bytes
Converting sectors 143394 to 143458 in track 8 (60.62% done)DEBUG (Aaru Format plugin): Memory snapshot: 295550704 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 312778152 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 312786344 bytes
Converting sectors 147490 to 147554 in track 8 (62.35% done)DEBUG (Aaru Format plugin): Memory snapshot: 330368504 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 347687536 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 347695728 bytes
Converting sectors 151586 to 151650 in track 8 (64.08% done)DEBUG (Aaru Format plugin): Memory snapshot: 369397696 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 386789656 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 386789656 bytes
Converting sectors 155682 to 155746 in track 8 (65.81% done)DEBUG (Aaru Format plugin): Memory snapshot: 404294344 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 421088640 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 421096832 bytes
Converting sectors 159765 to 159829 in track 9 (67.54% done)DEBUG (Aaru Format plugin): Memory snapshot: 438648520 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 455284168 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 455284168 bytes
Converting sectors 163861 to 163925 in track 9 (69.27% done)DEBUG (Aaru Format plugin): Memory snapshot: 477003640 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 494047296 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 494055488 bytes
Converting sectors 167957 to 168021 in track 9 (71.00% done)DEBUG (Aaru Format plugin): Memory snapshot: 511559056 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 528737728 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 528737728 bytes
Converting sectors 172053 to 172117 in track 9 (72.73% done)DEBUG (Aaru Format plugin): Memory snapshot: 546267680 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 563348408 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 563348408 bytes
Converting sectors 176137 to 176201 in track 10 (74.46% done)DEBUG (Aaru Format plugin): Memory snapshot: 265278944 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 280891800 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 280899992 bytes
Converting sectors 180243 to 180307 in track 11 (76.19% done)DEBUG (Aaru Format plugin): Memory snapshot: 298447320 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 314966496 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 314974688 bytes
Converting sectors 184339 to 184403 in track 11 (77.92% done)DEBUG (Aaru Format plugin): Memory snapshot: 336676680 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 353248520 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 353256712 bytes
Converting sectors 188435 to 188499 in track 11 (79.66% done)DEBUG (Aaru Format plugin): Memory snapshot: 370753184 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 288718360 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 288718360 bytes
Converting sectors 192531 to 192595 in track 11 (81.39% done)DEBUG (Aaru Format plugin): Memory snapshot: 308367448 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 324718408 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 324718408 bytes
Converting sectors 196627 to 196691 in track 11 (83.12% done)DEBUG (Aaru Format plugin): Memory snapshot: 346599512 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 363674784 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 363682976 bytes
Converting sectors 200723 to 200787 in track 11 (84.85% done)DEBUG (Aaru Format plugin): Memory snapshot: 381378624 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 398497000 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 398497000 bytes
Converting sectors 204802 to 204866 in track 12 (86.58% done)DEBUG (Aaru Format plugin): Memory snapshot: 420314328 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 432927072 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 432927072 bytes
Converting sectors 208898 to 208962 in track 12 (88.31% done)DEBUG (Aaru Format plugin): Memory snapshot: 454622080 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 470093240 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 470093240 bytes
Converting sectors 212994 to 213058 in track 12 (90.04% done)DEBUG (Aaru Format plugin): Memory snapshot: 487605768 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 503790296 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 503790296 bytes
Converting sectors 217090 to 217154 in track 12 (91.77% done)DEBUG (Aaru Format plugin): Memory snapshot: 525547448 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 541584968 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 541584968 bytes
Converting sectors 221186 to 221250 in track 12 (93.50% done)DEBUG (Aaru Format plugin): Memory snapshot: 559167128 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 575661864 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 575661864 bytes
Converting sectors 225282 to 225346 in track 12 (95.23% done)DEBUG (Aaru Format plugin): Memory snapshot: 593176888 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 610442232 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 610442232 bytes
Converting sectors 229378 to 229442 in track 12 (96.96% done)DEBUG (Aaru Format plugin): Memory snapshot: 632196168 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 649703832 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 649703832 bytes
Converting sectors 233474 to 233538 in track 12 (98.70% done)DEBUG (Aaru Format plugin): Memory snapshot: 667290088 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 679631280 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 679631280 bytes
Converting sectors 236482 to 236546 in track 12 (99.97% done)DEBUG (Aaru Format plugin): Memory snapshot: 695110488 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 710749952 bytes
DEBUG (Aaru Format plugin): Memory snapshot: 710758144 bytes
Converting sectors 236560 to 236560 in track 13 (100.00% done)
Converting tag CdTrackFlags in track 13 (100.00% done).
Written dump hardware list to output image.
Closing output image.

Conversion done.
```

**Comment by claunia** [comment]
`2022-01-20T23:53:04 Disc contains a hidden track...`

CDRWin does not support hidden tracks and other software that use the format may or may not only support hidden tracks of the same type.

Until we can be sure. this kind of images should not be convertible to the CDRWin cuesheet format.

See feature request #766

**Comment by claunia** [comment]
Fixed in 70d8299.
